src/Components/SearchBar/style.jsx

```diff
@@ -0,0 +1,55 @@
+import styled from 'styled-components';
+import { Search } from '@styled-icons/bootstrap/Search';
+
+export const Ssearchbar = styled.div`
+  padding-top: 18px;
+  @media (max-width: 1400px) {
+    padding-top: 0;
+  }
+  @media (max-width: 768px) {
+    height: 50px;
+    margin-bottom: 5px;
+  }
+`;
+
+export const Form = styled.form`
+  border: solid 2px var(--color--primary);
+  max-height: 40px;
+  min-height: 40px;
+  display: flex;
```

Comment:
Normally with 'display: flex' your items are online. So I don't see the use of 'flex-direction: row'